src/MomentoCacheClient.php

```diff
@@ -322,17 +322,10 @@ public function del(array|string $key, string ...$other_keys): Redis|int|false
         } else {
             $keys = array_merge([$key], $other_keys);
         }
-        $deletedKeys = 0;
         foreach ($keys as $key) {
-            $getResult = $this->client->get($this->cacheName, $key);
-            if ($getResult->asHit()) {
-                $deleteResult = $this->client->delete($this->cacheName, $key);
-                if ($deleteResult->asSuccess()) {
-                    $deletedKeys++;
-                }
-            }
+            $this->client->delete($this->cacheName, $key);
```

Comment:
Should also check the response and if false, return false immediately.

Reply:
addressed in this commit - b923807527f47540ce0dfb4e2a5acc94de5779ea